mongo/tests/conftest.py

```diff
@@ -2,11 +2,17 @@
 # All rights reserved
 # Licensed under a 3-clause BSD style license (see LICENSE)
 import copy
+import itertools
+import json
 import os
 import time
+from functools import partial
```

Comment:
isn't this breaking `isort` check ?

Reply:
Why would that be?

Reply:
Isn't `isort` sorting imports alphabetically ? I would have expected it to warn `functools` being after `time`

Reply:
See https://www.python.org/dev/peps/pep-0008/#imports
1st group is the standard library
2nd group are third-party libraries
3rd group are local application

Reply:
ah thanks!